keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -561,11 +549,14 @@ class ClusterHandler(
   /**
    * @return true if the only difference is in the onlyEnabledServerGroup property
    */
-  private fun ResourceDiff<ServerGroup>.isEnabledOnly(): Boolean =
+  fun ResourceDiff<ServerGroup>.isEnabledOnly(): Boolean =
     current != null &&
       affectedRootPropertyNames.all { it == "onlyEnabledServerGroup" } &&
       current!!.onlyEnabledServerGroup != desired.onlyEnabledServerGroup
 
+  override fun isDiffOnlyInEnabled(diff: ResourceDiff<ServerGroup>): Boolean =
+    diff.isEnabledOnly()
```

Comment:
I'm confused why there's an extension function and then an abstract method whose implementations are identical and just call the extension function. Why not just call that extension function directly?

Reply:
Well, my thought was that I didn't know how to stub or mock extension functions. But, since I am not doing that for this guy I should do I the other way.